.github/workflows/release.yaml

```diff
@@ -31,9 +31,9 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: true
-      matrix:
+      matrix: # do we need to define a matrix here, normally you will have a single docker file
```

Comment:
Falkor is using multiple dockerfile in their workflow. If users wanna build multi-resource service, they might need it. https://github.com/FalkorDB/falkordb-omnistrate/blob/d5200b948382e4df7e44c4ece05389307d5313e4/.github/workflows/build-release-image.yaml#L26